index.html

```diff
@@ -60,40 +61,56 @@ <h1 class="visually-hidden">Город Седона</h1>
         <h2 class="visually-hidden">Преимущества:</h2>
         <ol class="advantages-list">
           <li class="advantages-item advantages-item-large">
-            <h3 class="advantages-item-title">Настоящий городок</h3>
-            <p class="advantages-item-content">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
-            <img src="images/real-city.jpg" width="800" height="384" alt="Городок Седона">
-            <ul class="adviсes-list">
-              <li class="adviсes-item advices-item-housing">
-                <h4 class="adviсes-item-title">Жильё</h4>
-                <p class="advices-item-content">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+            <div class="large-wrapper advantages-item-wrapper">
+              <h3 class="advantages-item-title">Настоящий городок</h3>
+              <p class="advantages-item-content">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+            </div>
+            <img class="advantages-item-img" src="images/real-city.jpg" width="800" height="384" alt="Городок Седона">
+            <ul class="advices-list">
+              <li class="advices-item advices-item-housing">
+                <div class="advises-item-wrapper">
```

Comment:
```suggestion
                <div class="advices-item-wrapper">
```

Только русскую не поставь =)